src/main/java/com/commercetools/helper/mapper/impl/payment/BasePaymentMapperImpl.java

```diff
@@ -253,6 +253,11 @@ protected Item createPaypalPlusItem(@Nonnull LocalizedString itemName, @Nonnull
         return createPaypalPlusItem(itemName, locales, dlipfq.getQuantity(), dlipfq.getDiscountedPrice().getValue());
     }
 
+    private Item plusSkuProductName(@Nonnull LineItem lineItem, @Nonnull List<Locale> locales, @Nonnull Item targetItem) {
```

Comment:
Maybe to call variable `pppItem` or `paypalItem` would be better? See naming also here: https://github.com/commercetools/commercetools-paypal-plus-integration/pull/143/files#diff-98208488d399ffeb7c0443fb763ae32aR261

Reply:
Renamed the variable to `paypalItem`
https://github.com/commercetools/commercetools-paypal-plus-integration/pull/143/commits/0a1b6827d4385c563489ad27efdfb7dee88bc706